Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -32,6 +32,7 @@
 NSString * const BRANCH_PREFS_KEY_USER_URL = @"bnc_user_url";
 NSString * const BRANCH_PREFS_KEY_IS_REFERRABLE = @"bnc_is_referrable";
 NSString * const BRANCH_PREFS_KEY_BRANCH_UNIVERSAL_LINK_DOMAINS = @"branch_universal_link_domains";
+NSString * const BRANCH_REQUEST_KEY_EXTERNAL_INTENT_URI = @"external_intent_uri";
```

Comment:
this does not compile for me because this symbol is defined 2x (see also https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK/pull/259/files#diff-05f7881939e7f659c301d3248c960442R63)